plugins/outputs/otlp/client_test.go

```diff
@@ -0,0 +1,81 @@
+package otlp
+
+import (
+	"context"
+	"net/url"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	metricsService "go.opentelemetry.io/proto/otlp/collector/metrics/v1"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func TestClientWithRecoverableError(t *testing.T) {
+	listener := newLocalListener()
+	grpcServer := grpc.NewServer()
+	mockMetricsServer := metricServiceServer{
+		status: nil,
+	}
+	metricsService.RegisterMetricsServiceServer(grpcServer, &mockMetricsServer)
+	go func() {
+		_ = grpcServer.Serve(listener)
```

Comment:
Why do you ignore the returned error? I you really need to ignore it, please add a comment. Just silencing the linter is not a good reason. :-)

Reply:
No longer ignoring it.